### ISSUE TITLE
The font is nice, but bad for accessibility

### ISSUE THREAD

**Issue by SkeletonTree485** [SkeletonTree485/NoOneOnline#1]
the font (at least in paragraphs) should be replaced with something more readable as currently it is terrible for accessibility
